server/src/main/java/org/elasticsearch/client/internal/ClusterAdminClient.java

```diff
@@ -456,6 +459,10 @@ public interface ClusterAdminClient extends ElasticsearchClient {
      */
     RestoreSnapshotRequestBuilder prepareRestoreSnapshot(String repository, String snapshot);
 
+    void snapshotGlobalState(SnapshotGlobalStateRequest request, ActionListener<SnapshotGlobalStateResponse> listener);
+
+    SnapshotGlobalStateRequestBuilder prepareGlobalStateSnapshot(String repository, String snapshot);
+
```

Comment:
These helper methods are a legacy of the old transport client, no need to add them for new actions - just execute the action directly.

```suggestion
```

Reply:
Removed this block of code as no more listening requests through `AdminClient`.